### DISCUSSION TITLE
First note starting on "currentTime+1" : it fails

### DISCUSSION THREAD

**Discussion by rogerio-saldanha** [danigb/smplr#75] [Q&A] [answered]
Hi,
First of all, thank you very much for this package! :-)
I may have found a bug. I had to work around it.
I'm using:
    "smplr": "^0.13.3"
My code:

const context = new AudioContext()
const piano = await new SplendidGrandPiano(context).load
const now = context.currentTime
piano.start({ note: "C0", velocity: 1, time: 0, duration: 0.01 })   // Dummy note, not to be heard.
piano.start({ note: "A4", velocity: 80, time: now + 1, duration: 2 })

If I don't create the dummy "C0", then "A4" will not be heard.
With the dummy "C0" (not to be heard), then "A4" sounds fine.
Is this expected behavior?

Thanks and regards

**Comment by danigb** [comment, marked as answer]
This works for me:
```html
<html>
  <body>
    <button id="btn">play</button>
  </body>
  <script type="module">
    import { SplendidGrandPiano } from "https://unpkg.com/smplr/dist/index.mjs"; 
    const context = new AudioContext(); 
    const piano = new SplendidGrandPiano(context); 

    document.getElementById("btn").onclick = () => {
      context.resume(); // enable audio context after a user interaction
      const now = context.currentTime
      piano.start({ note: 60, velocity: 80, time: now + 1 }); // play the note after 1 second
    };
  </script>
</html>
```
So I guess the problem is that you didn't resume the audio context first.

**Reply by rogerio-saldanha** [reply]
Cool :-) solved my issue.
Thank you!
